sdk/search/search-documents/src/generated/service/models/index.ts

```diff
@@ -992,7 +1064,7 @@ export interface GetIndexStatisticsResult {
    * The amount of memory in bytes consumed by vectors in the index.
    * NOTE: This property will not be serialized. It can only be populated by the server.
    */
-  readonly vectorIndexSize?: number;
+  readonly vectorIndexSize: number;
```

Comment:
If this type has already been released does adding a new required property break anything?

Reply:
This would only break users who explicitly instantiate an object of this type with this field omitted. As it's a response-only type, I don't see this being an issue.